apis/Google.Cloud.Iam.V1/Google.Cloud.Iam.V1/PolicyPartial.cs

```diff
@@ -77,5 +89,8 @@ public bool RemoveRoleMember(string role, string member)
         }
 
         private Binding FindRole(string role) => Bindings.FirstOrDefault(binding => binding.Role == role);
+
+        private void ValidatePolicyVersion() =>
+            GaxPreconditions.CheckState(Version <= 1, "Helper methods cannot be invoked on policies with version {0}. For more information, see FIXME.", Version);
```

Comment:
How is `Version` set in this case?

Reply:
This is just a partial class for the proto - so it's "whatever the proto property has been set to". That could be in client side code for whatever reason, or a policy resource returned by a server.